polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -493,6 +501,8 @@ impl ApprovalVotingSubsystem {
 			metrics,
 			Arc::new(SystemClock {}),
 			spawner,
+			MAX_APPROVAL_RETRIES,
+			APPROVAL_CHECKING_TIMEOUT / 2,
```

Comment:
Looking at this constant, I can see it was introduced in https://github.com/paritytech/polkadot/pull/3306, but don't recall what was the purpose of it and of `approval_outcome` (apart from `Approved`) which we don't seem to even log.

What is the rationale for setting the retry_backoff to 1 minute?

Reply:
It can't be higher than `APPROVAL_CHECKING_TIMEOUT` because of this: https://github.com/paritytech/polkadot-sdk/blob/63c73bf6db1c8982ad3f2310a40799c5987f8900/polkadot/node/core/approval-voting/src/lib.rs#L755.

> What is the rationale for setting the retry_backoff to 1 minute?

1 minute seem to satisfy the following requirements for me, it is rare enough to not create too much load and give time for block to be finalized because of no-shows cover-up, it is small enough to actually help with the finality in case the no-show from this particular node is the one holding the approval.